wct.conf.json

```diff
@@ -0,0 +1,15 @@
+{
+  "plugins": {
+    "local": {
+      "browserOptions": {
+        "chrome": [
+          "headless",
```

Comment:
Wait... we can do this?! @notwaldorf

Reply:
that's what @azakus says and he's the man

Reply:
Omg! You don't know how excited I am! I was just off by a bit on figuring this out on my own a month ago

Reply:
You'll need WCT 6.5+, as the argument pass-through was just landed.

Reply:
Would love to see Tedium convert all the Travis configs to use that!